libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -58,7 +59,11 @@ void SRV_Channel::output_ch(void)
                 // non-mapped rc passthrough
                 int16_t radio_in = c->get_radio_in();
                 if (passthrough_mapped) {
+                  if (rc().in_rc_failsafe() && ((1U<<passthrough_from) & SRV_Channels::get_rc_fs_mask())) {
+                   radio_in = c->get_radio_trim();
```

Comment:
```radio_in = servo_trim;```
Presumably you want to go to the servo trim not the radio trim?

Reply:
I don't think so...control is an input thing

Reply:
This currently goes direct to the RC input trim in PWM. It a scaled passthrough, that bypasses the scaling. With this PR RC input valid at trim will be a different value to invalid at trim. What you want is `pwm_from_angle(0 * 4500)` which is the same as `pwm_from_angle(0)` which is the same as `servo_trim`.

Reply:
There is no guarantee the radio trim is even within the configured output range of the servo.

Reply:
yep...fixed